download-chromedriver.js

```diff
@@ -2,7 +2,15 @@ const fs = require('fs')
 const path = require('path')
 const electronDownload = require('electron-download')
 const extractZip = require('extract-zip')
-const versionToDownload = require('./package').version
+
+function getVersion () {
+  const pack = require('./package') || {}
+  const dependencies = pack.dependencies
+  const devDependencies = pack.devDependencies
+  const allDependencies = { ...dependencies, ...devDependencies }
+
+  return allDependencies['electron'] || pack.version
```

Comment:
This won't work when the electron version is a caret value, eg `^7.0.0-beta.5` as electron-download doesn't know how to deal with semver caret ranges.